pkg/validator/validator.go

```diff
@@ -20,12 +20,15 @@ import (
 	"bytes"
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"net/http"
 	"net/url"
 	"path"
 	"time"
+
+	validationError "github.com/abcxyz/jvs-plugin-jira/pkg/errors"
```

Comment:
I question the necessity of putting validator in a separate package. How about we move validator to the `plugin` package and just define the "invalid" error as a local error.

Reply:
Done